MediaBrowser.Model/System/SystemStorageInfo.cs

```diff
@@ -0,0 +1,62 @@
+#nullable disable
+#pragma warning disable CS1591
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using MediaBrowser.Model.Updates;
+
+namespace MediaBrowser.Model.System;
+
+/// <summary>
+/// Contains informations about the systems storage.
+/// </summary>
+public class SystemStorageInfo
+{
+    /// <summary>
+    /// Gets or sets the program data path.
+    /// </summary>
+    /// <value>The program data path.</value>
+    public FolderStorageInfo ProgramDataDirectoryInfo { get; set; }
```

Comment:
If all the variables in here are just information about paths the "DirectoryInfo" suffix is unneeded. (Same for all other variables too)

```suggestion
    public FolderStorageInfo ProgramData { get; set; }
```

If we chose to generalize this class (E.g. `SystemResourcesDto`) then we do want some suffix, but I think DirectoryInfo is quite long. Maybe just `Folder` or `Storage` works, also consistent with the "FolderStorageInfo" class name it uses.

```suggestion
    public FolderStorageInfo ProgramDataFolder { get; set; }
```

```suggestion
    public FolderStorageInfo ProgramDataStorage{ get; set; }
```